tests/test-sessions/test-jdbc-sessions/src/test/java/org/eclipse/jetty/server/session/JdbcTestHelper.java

```diff
@@ -303,15 +305,15 @@ public static void insertSession (String id, String contextPath, String vhost,
             
             if (attributes != null)
             {
+                SessionData tmp = new SessionData (id, contextPath, vhost, created, accessed, lastAccessed, maxIdle);
                 ByteArrayOutputStream baos = new ByteArrayOutputStream();
                 ObjectOutputStream oos = new ObjectOutputStream(baos);
-                Map<String,Object> emptyMap = Collections.emptyMap();
-                oos.writeObject(emptyMap);
+                SessionData.serializeAttributes(tmp, oos);
```

Comment:
`ObjectOutputStream` not closed, use try-with-resources.

Reply:
ok